mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_sum_bf16.mlir

```diff
@@ -29,7 +30,7 @@
 // Do the same run, but now with  VLA vectorization.
 // RUN: %if mlir_arm_sve_tests %{ %{compile_sve} | %{run_sve} | FileCheck %s %}
 
-// UNSUPPORTED: target=aarch64{{.*}}
+// UNSUPPORTED: target=aarch64{{.*}}, mlir_arm_emulator
```

Comment:
Why is `mlir_arm_emulator`? Could you add a note in the summary? Thanks!

Reply:
I added comments in mlir/test/lit.site.cfg.py.in and the summary.